apps/hash-ai-worker-ts/src/activities/shared/get-llm-response/get-anthropic-response.ts

```diff
@@ -87,6 +87,18 @@ const convertAnthropicRateLimitRequestsResetTimestampToMilliseconds = ({
   return rateLimitEndInMilliseconds;
 };
 
+const throttledStartingDelay = 15_000; // 15 seconds
+
+const isErrorAnthropicThrottlingError = (error: unknown): boolean =>
+  error instanceof APIError &&
+  !!error.status &&
+  /**
+   * @todo: This is a temporary solution until we have a better way to
+   * determine if the error is caused by throttling.
+   */
+  error.status >= 500 &&
+  error.status < 600;
```

Comment:
Technically two attributable/known errors are being caught up here, aswell:

- `500` - `api_error`: An unexpected error has occurred internal to Anthropic’s systems.
- `529` - `overloaded_error`: Anthropic’s API is temporarily overloaded.

Whereas the `520` we're experiencing appears to be originating with Anthropic's use of Cloudflare, and isn't an error code they [profess to throw](https://docs.anthropic.com/en/api/errors) themselves.

That said, I think these two known errors can be handled in the same way (with a jittered retry).